site/_includes/explanations/partitioned-topic-admin.md

```diff
@@ -1,4 +1,4 @@
-You can use Pulsar's [admin API](../../admin/AdminInterface) to create and manage partitioned topics.
+You can use Pulsar's [admin API](../../admin-api/overview) to create and manage partitioned topics.
```

Comment:
yes, right we can keep overview-link for for admin-api text. may be we can add link on `partitioned topics` as `partitioned-topics`. it's not necessary but it might be useful for user.